IPy.py

```diff
@@ -1383,6 +1383,8 @@ def parseAddress(ipstr):
     >>> testParseAddress('::FFFF:129.144.52.38')
     281472855454758 (IPv6)
     """
+    """ Make sure there is no space in `ipstr`, if there is raise `ValueError`."""
+    if " " in ipstr: raise ValueError("IP Address format was invalid: %s" % ipstr)
```

Comment:
What I was trying to illustrate is that any whitespace is removed and a valid address is accepted. To be consistent, you would have to reject any whitespace.

Meanwhile, I don't understand the value in rejecting it. If you're attempting to validate an address, I'd use the output of the function as the validated address rather than comparing it.